### PR TITLE
Update logo section & spelling / grammar

### PR DESCRIPTION
**Pull request by srleach**
## Features

- Add logo section
- Amend some spelling mistakes

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -325,9 +333,10 @@ Client-side JavaScript or API
```
Comment (wnowicki) [inline index.md]: In here I'd reword whole section, driving retailer more to API integration. And only small mention of HTML form.

File: index.md [hunk 2]
```diff
@@ -77,6 +77,14 @@ have completed all activities below:
...
 3. Compliant finance promotion
 4. Mandatory finance information page
+5. (Optional) Your logo to be displayed in the checkout process.
+
+#### Your Logo
+
+Your logo should be of a sufficient resolution to be displayed accross a standard range of monitors. 
+It should have a transparent background, but white is acceptable if transparent is not possible.
+ 
+It must be of a minimum resolution (220 x 80) - but ideally should be bigger.
```
Comment (wnowicki) [inline index.md]: Have we confirmed this with code? Do we know max size as well?

Comment (srleach) [inline index.md]: Not in code directly, I've taken the example size of the shopto logo from S3 and followed that as a guide.

We're able to take any size as long as it follows this aspect ratio, the browser will downsize. We just need to be aware of the image size when we add to s3 and downsize if necessary.

**Comment by srleach** [comment]
@wnowicki - I'm going to have a go at rewriting a swathe of this particular page. Don't merge yet, I'll open PR early next week with it :+1:

File: index.md (hunk 2 above)
Comment (wnowicki) [inline index.md]: From where you've taken aspect ratio? What if Retailer has 1:1? For now as we don't have any rules around logo, we have to always agree that for each retailer. 

Please don't guess ;)

**Comment by wnowicki** [comment]
Two options here we are trying to finish it COB today. Or we are moving some changes to https://github.com/PayBreak/retailer-integration-guide/tree/retailer-feedback and closing this one. 

As me if you are not sure. I want to clear PR's queue a little bit.

**Comment by srleach** [comment]
Lets close PR. There's much we can change with both parts of what I've touched here - so lets agree on _what_ i will touch and i'll update this in a new PR after a rewrite of the bits we choose to do?

**Comment by srleach** [comment]
Or, advise what you would like to change and I'll rewrite those sections today before COB?

**Comment by srleach** [comment]
@wnowicki Ready
